cmd/query.go

```diff
@@ -114,6 +107,10 @@ func runQueryCmd(cmd *cobra.Command, args []string) {
 	// convert the query or sql file arg into an array of executable queries - check names queries in the current workspace
 	queries := execute.GetQueries(args, workspace)
 
+	// start db if necessary
+	err = db.EnsureDbAndStartService(db.InvokerQuery)
```

Comment:
why not in prerun as for other commands?

Reply:
fixed.

commented previous code for now